### ISSUE TITLE
about train loss

### ISSUE THREAD

**Issue by lin076** [ali-vilab/UniAnimate-DiT#49]
Hi, Thanks for your great work! I’ve been training the model based on your code, and I’m wondering — in your experiments, what was the typical range of training loss when the model is considered reasonably trained?
Is there a rough threshold or curve shape I should expect for convergence? Any reference value would be really helpful for monitoring and early stopping.Thanks again!

**Comment by pooyafayyazs** [comment]
I'm fine-tuning the model the seems the loss is just fluctuating, I did not see any loss curve.
